bin/run-browser-test-env

```diff
@@ -118,22 +85,20 @@ compose_command="${compose_command} up"
 
 readonly START_EMULATOR_COMMAND="${compose_command} -d ${emulator}"
 echo "Running docker compose to start the cloud emulator:"
-echo "${START_EMULATOR_COMMAND}"
+echo "  ${START_EMULATOR_COMMAND}"
```

Comment:
is this space intentional

Reply:
Yes, makes it a little easier to scan the output and see the line with the command as related to the one above it